netbox/templates/dcim/macaddress.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'generic/object.html' %}
+{% load helpers %}
+{% load plugins %}
+{% load render_table from django_tables2 %}
+{% load i18n %}
+
+{% block content %}
+  <div class="row">
+    <div class="col col-md-4">
+      <div class="card">
+        <h2 class="card-header">{% trans "MAC Address" %}</h2>
+        <table class="table table-hover attr-table">
+          <tr>
+            <th scope="row">{% trans "Description" %}</th>
+            <td>{{ object.description|placeholder }}</td>
+          </tr>
+          <tr>
+            <th scope="row">{% trans "Assignment" %}</th>
+            <td>
+              {% if object.assigned_object %}
+                {% if object.assigned_object.parent_object %}
+                  {{ object.assigned_object.parent_object|linkify }} /
+                {% endif %}
```

Comment:
`parent_object` should always be a device or VM; I think it's ok to remove the conditional.